### PR TITLE
Adaptive SMC implementation + MonteCarloMeasurements integration

### PR DESCRIPTION
**Pull request by francescoalemanno**

### CODE REVIEW

**Comment by francescoalemanno** [comment]
@baggepinnen i would like to merge this PR, where i reexport your package `MonteCarloMeasurements` symbols for seamless integration, what do you think about this, is it okay for you?

btw it is a spectacular package

**Comment by baggepinnen** [comment]
Cool, you have my blessing 😃
If you give me a few hours I can look it through and see if there is any tips I can contribute with before you pull

File: src/KissABC.jl [hunk 1]
```diff
@@ -155,7 +110,7 @@ end
 
 # Generalities
 
-This function will run an Affine Invariant MCMC sampler, and will return an `AISChain` object with all the parameter samples,
+This function will run an Affine Invariant MCMC sampler, and will return an `Particles` object for each parameter,
```
Comment (baggepinnen) [inline src/KissABC.jl]:
```suggestion
This function will run an Affine Invariant MCMC sampler, and will return a `Particles` object for each parameter,
```

**Review by baggepinnen — APPROVED** [review]
I had a look through the code and nothing appears to stand out to me :) I'm happy you find MCM useful!

**Comment by francescoalemanno** [comment]
Thank you!

**Pull request merged**